.envrc

```diff
@@ -1,4 +0,0 @@
-if ! has nix_direnv_version || ! nix_direnv_version 2.3.0; then
-    source_url "https://raw.githubusercontent.com/nix-community/nix-direnv/2.3.0/direnvrc" "sha256-Dmd+j63L84wuzgyjITIfSxSD57Tx7v51DMxVZOsiUD8="
-fi
-use flake
```

Comment:
> Both nix-direnv and legacy command nix-shell fail. Reverting to flakes-only commands.

Nix-direnv *does* use flakes, as `use flake` in the `.envrc` shows. I would like it not to be removed, though, it should probably be updated to version 3.0.4.

Reply:
Reverted to https://github.com/space-wizards/space-station-14/pull/25361/commits/847c80434c1de6ec8feb52cf25c531a9f4b39144